docs/docs-beta/docs/dagster-plus/deployment/environment-variables/dagster-ui.md

```diff
@@ -5,4 +5,115 @@ sidebar_position: 1
 sidebar_label: "Set with Dagster+ UI"
 ---
 
-# Set environment variables with the Dagster+ UI
\ No newline at end of file
+Environment variable are key-value pairs that are set outside of your source code. Using environment variables lets you dynamically change the behavior of your application without modifying source code and securely set up secrets.
+
+Dagster supports several approaches for [accessing environment variable in your code](/todo). You can also set environment variables in several ways. This guide will cover how to set environment variables in the Dagster+ UI.
+
+## What you'll learn
+
+- How to add, edit, and delete environment variables in the Dagster+ UI
+- How to modify the value of environment variables based on the deployment environment
+- How to export environment variables for local use
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- **Organization Admin**, **Admin**, or **Editor** permissions for your Dagster+ account
+- To be using Dagster version 1.0.17 or later
+
+</details>
+
+## How to find the environment variables page
+1. Sign in to your Dagster+ account
+2. Use the deployment switcher in the top right corner to select the deployment where you want to view or manage environment variables
+3. Navigate to the **Deployment** -> **Environment variables** page
+
+## How to add an environment variable
+To create a new environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **+ Add environment variable** button
+3. A modal will open, fill out the following information
+    - **Name** - Enter a name for the environment variable. This is how the variable will be referenced in your code.
+    - **Value** - Enter a value for the environment variable.
+    - **Deployment Scope** - select the deployment(s) where the variable should be accessible:
+        - **Full deployment** - The variable will be available to selected code locations in the full deployment
+        - **Branch deployments** - The variable will be available to selected code locations in Branch Deployments
+        - **Local** - If selected, the variable will be included when [exporting environment variables to a local `.env` file](#how-to-export-environment-variables-locally)
+    - **Code Location Scope** - select the code location(s) where the variable should be accessible. At least one code location is required.
+
+<!-- TODO SCREENSHOT -->
+
+4. Click **Save**
+
+## How to edit an environment variable
+To edit an existing environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **Edit** button on the right of the environment variable you want to edit
+3. A modal will open where you can edit the attributes of the environment variable
```

Comment:
i like that! I think i'm going to keep the longer instructions for downloading the .env file because i got tripped up on how to do that and actually load the values when i run dagster locally. otherwise, most of these are really self-explanatory